README.md

```diff
@@ -2,29 +2,46 @@
 # AAVE Starknet Bridge
 
 [![Tests](https://github.com/aave-starknet-project/aave-starknet-bridge/actions/workflows/ci.yml/badge.svg)](https://github.com/aave-starknet-project/aave-starknet-bridge/actions/workflows/ci.yml)
-[![GitHub pull requests](https://img.shields.io/github/issues-pr/cdnjs/cdnjs.svg?style=flat)](https://github.com/aave-starknet-project/aave-starknet-bridge/pulls)
-[![Issues](https://img.shields.io/github/issues-raw/tterb/PlayMusic.svg?maxAge=25000)](https://github.com/aave-starknet-project/aave-starknet-bridge/issues)
-
- <a href="https://github.com/aave-starknet-project/aave-starknet-bridge/blob/main/LICENSE.md"><img src="https://img.shields.io/github/license/sourcerer-io/hall-of-fame.svg?colorB=ff0000"></a>
+[![Tests](https://github.com/aave-starknet-project/aave-starknet-bridge/actions/workflows/deploy.yml/badge.svg)](https://github.com/aave-starknet-project/aave-starknet-bridge/actions/workflows/deploy.yml)
+<a href="https://github.com/aave-starknet-project/aave-starknet-bridge/blob/main/LICENSE.md"><img src="https://img.shields.io/github/license/sourcerer-io/hall-of-fame.svg?colorB=ff0000"></a>
 
 :warning: This codebase is still in an experimental phase, has not been audited, might contain bugs and should not be used in production.
 
+
+## Table of contents
```

Comment:
Thanks a lot for this! We could add numbers and group some sections together, like this:

```
## Table of contents
  * [1. Overview](#overview)
  * [2. Architecture](#architecture)
    * [2.1. Contracts](#contracts)
    * [2.2. ETHStaticATokens on L2](#ethstaticatokens-on-l2)
    * [2.3. Bridging aTokens from L1<>L2](#bridging-atokens-from-l1--l2)
      * [2.3.1. Transfer L1->L2](#transfer-l1--l2-)
      * [2.3.2. Transfer L2->L1](#transfer-l2--l1-)
    * [2.4. Synchronisation of rewards on L1 <> L2](#synchronisation-of-rewards-on-l1----l2)
    * [2.5. Claiming rewards on L2](#claiming-rewards-on-l2)
    * [2.6. Bridging rewards from L2->L1](#bridging-rewards-from-l2--l1)
    * [2.7. Proxies](#proxies)
    * [2.8. Governance](#governance)
  * [3. Development](#governance)
    * [3.1 Setup](#environment)
    * [3.2 Tests](#run-the-tests)
    * [3.3. Deployment](#deployment)
```

Reply:
agreed